core/src/mindustry/client/Client.java

```diff
@@ -44,32 +47,34 @@
     public static boolean hideTrails = true;
     public static Ratekeeper configRateLimit = new Ratekeeper();
     public static boolean hideUnits = false;
-    /** The last position someone sent in chat or was otherwise put into the buffer, in tile coords. */
+    /**
+     * The last position someone sent in chat or was otherwise put into the buffer, in tile coords.
+     */
     private static Vec2 lastSentPos = new Vec2();
     public static IntSet messageBlockPositions = new IntSet();
     public static final String messageCommunicationPrefix = "IN USE FOR CHAT AUTHENTICATION, do not use";
 
     public static void initialize() {
         fooCommands.<Player>register("help", "[page]", "Lists all client commands.", (args, player) -> {
-            if(args.length > 0 && !Strings.canParseInt(args[0])){
+            if(args.length > 0 && !Strings.canParseInt(args[0])) {
                 player.sendMessage("[scarlet]'page' must be a number.");
                 return;
             }
             int commandsPerPage = 6;
             int page = args.length > 0 ? Strings.parseInt(args[0]) : 1;
-            int pages = Mathf.ceil((float)fooCommands.getCommandList().size / commandsPerPage);
+            int pages = Mathf.ceil((float) fooCommands.getCommandList().size / commandsPerPage);
 
-            page --;
+            page--;
 
-            if(page >= pages || page < 0){
+            if(page >= pages || page < 0) {
                 player.sendMessage("[scarlet]'page' must be a number between[orange] 1[] and[orange] " + pages + "[scarlet].");
                 return;
             }
 
             StringBuilder result = new StringBuilder();
-            result.append(Strings.format("[orange]-- Client Commands Page[lightgray] @[gray]/[lightgray]@[orange] --\n\n", (page+1), pages));
+            result.append(Strings.format("[orange]-- Client Commands Page[lightgray] @[gray]/[lightgray]@[orange] --\n\n", (page + 1), pages));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.